api/tests/routes/pro/patch_venue_test.py

```diff
@@ -578,6 +580,282 @@ def test_should_not_update_opening_hours_with_lower_case_weekday(self, client):
             }
         }
 
+    @patch("pcapi.core.search.async_index_offers_of_venue_ids")
+    def when_changes_on_public_name_algolia_indexing_is_triggered(self, mocked_async_index_offers_of_venue_ids, client):
```

Comment:
Il me semble que les méthodes de test doivent commencer par `test_` pour être effectivement lancées.
Ici et pour quelques suivants.
```suggestion
    def test_when_changes_on_public_name_algolia_indexing_is_triggered(self, mocked_async_index_offers_of_venue_ids, client):
```

Reply:
Ils sont bien collectés par pytest, je pense parce qu'ils sont dans une classe qui finit par Test, mais j'ai changé par uniformité